info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompiler.java

```diff
@@ -272,6 +272,11 @@ private void storeElectionContestCandidateInDatabase(JsonObject candidate,
       List<Value<String>> candidateIds, List<Value<Boolean>> candidateIncumbency) {
     String name = candidate.get("name").getAsString();
     String party = candidate.get("party").getAsString();
+    String email = getField(candidate, "email");
+    String phoneNumber = getField(candidate, "phone");
+    String photo = getField(candidate, "PhotoUrl");
```

Comment:
nit. photoUrl. Also, you're using photoURL later on in line 293. It should be consistent